core/cmd/local_client.go

```diff
@@ -102,8 +102,8 @@ func (cli *Client) runNode(c *clipkg.Context) error {
 		if err = ldb.Close(); err != nil {
 			lggr.Criticalf("Failed to close LockedDB: %v", err)
 		}
-		if err = lggr.Sync(); err != nil {
-			log.Printf("Failed to sync Logger: %v", err)
+		if err = cli.Logger.Close(); err != nil {
```

Comment:
I need to look closer, but I think this races with the normal shutdown `Close()` call. Maybe should make it idempotent defensively, with a `sync.Once`?